components/collective-page/sections/Contribute.js

```diff
@@ -149,134 +150,181 @@ class SectionContribute extends React.PureComponent {
     const isEvent = collective.type === CollectiveType.EVENT;
     const hasContribute = collective.isActive || isAdmin;
     const hasOtherWaysToContribute = !isEvent && (isAdmin || events.length > 0 || subCollectives.length > 0);
+    const isActive = collective.isActive;
+    const hasHost = collective.host;
+
+    /*
+    cases
+
+    1. admin + no host = Contribute Section and 'Start accepting financial contributions' ✅
+    2a. admin + host = normal Contribute section ✅
+    2b. not admin + Collective active = normal Contribute section ???
+    3. not admin + Collective not active = display nothing ✅
+    */
 
     const createContributionTierRoute = isEvent
       ? `/${collective.parentCollective.slug}/events/${collective.slug}/edit#tiers`
       : `/${collective.slug}/edit/tiers`;
 
-    if (!hasContribute && !hasOtherWaysToContribute) {
+    if (!isAdmin && !isActive) {
```

Comment:
**[:sparkles: **Enhancement**]** One problem about this is that it is going to hide events for archived collectives and collectives not approved yet.

---

<details>
<summary>More details about this feedback</summary>
<br/>

This feedback categorization is inspired by [Netlify's Feedback Ladders system](https://www.netlify.com/blog/2020/03/05/feedback-ladders-how-we-encode-code-reviews-at-netlify). It includes 5 levels of feedback:

:rotating_light: **Critical blocker**
This feedback blocks all related work, and requires immediate action. 

:no_entry_sign: **Blocker**
This feedback should block the work from moving forward or being approved, but it does not necessarily require immediate action from the team.

:hammer_and_wrench: **Important**
This feedback should not block the work from moving forward or being approved, but it does require future action from the team. This generally means the feedback is not required for the MVP or initial draft of this work, but should be addressed at some point and we should ideally create an issue to track it.

:sparkles: **Enhancement**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they agree with it.

:ice_cream: **Details & personal preferences**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they deem it useful. This is also the category for everything that relates to personal preferences or code style.

</details>